lib/logging/lib/logging.c

```diff
@@ -112,15 +111,14 @@ logging_fini(void)
 }
 
 void
-log_impl(
-    const int level,        /* log level                     */
-    const char *const file, /* file                          */
-    const int lineno,       /* line number                   */
-    const char *const func, /* function name                 */
-    uint64_t *const timer,  /* timer                         */
-    const merr_t err,       /* error value                   */
-    const char *const fmt,  /* format string                 */
-    ...)                    /* variable-length argument list */
+log_impl(const int level,       /* log level                     */
```

Comment:
I think these comments are probably throwing this off a bit. Try removing them next time your format the code.

Reply:
The comment are not messing this up.  The indent is 8 due to `ContinuationIndentWidth`